data/ecosystem/vendors.yaml

```diff
@@ -209,6 +209,13 @@
   contact: ''
   oss: false
   commercial: true
+- name: qryn
+  distribution: true
+  nativeOTLP: true
+  url: 'https://qryn.cloud'
```

Comment:
Can you please update this with a link to your documentation that details how your offering consumes OpenTelemetry natively via OTLP?

Reply:
Hello @svrnm thanks for taking the time to review!

We support OTLP ingestion through collector writing to db as well as protocol consumers in our ingestion APIs:

- https://github.com/metrico/otel-collector
- https://qryn.metrico.in/#/support?id=tempo-api /v1/traces (OTLP)
- https://github.com/metrico/qryn/blob/b7c8645fbb308889c0fdebbb0e714abce94fec1c/parsers.js#L175

Let me know if the links provide sufficient information.

Reply:
This page is (currently) specifically about vendor's that support OTLP with their backend, so update your URL to the link that points to your ingestion API.

Reply:
Done, thanks!